scene/gui/color_picker.cpp

```diff
@@ -198,7 +198,7 @@ void ColorPicker::_html_entered(const String &p_html) {
 		return;
 
 	set_pick_color(color);
-	emit_signal("color_changed", color);
+	// emit_signal("color_changed", color);
```

Comment:
why was this line commented out?

Reply:
Commenting this line prevents it from printing the message twice when you exit ColorPicker

Reply:
Commenting this line prevents it from committing the selected color when you exit the ColorPicker

Reply:
is still the message printed when color is changed by entering html hex color then?

Reply:
Actually it ended up breaking the functionality... I uncommented it and rebuilt it and the result was this:

![A](https://user-images.githubusercontent.com/37383316/67248790-e8f30800-f43b-11e9-9d50-662eb5f8eac1.gif)

When you leave ColorPicker it save the last "modulate" that was changed. I believe this isn't a bug but a way for the engine to choose one of the two properties...